pkg/security/podsecuritypolicy/selinux/mustrunas_test.go

```diff
@@ -154,7 +161,9 @@ func TestMustRunAsValidate(t *testing.T) {
 			continue
 		}
 
-		errs := mustRunAs.Validate(nil, nil, nil, tc.podSeLinux)
+		internalSELinuxOptions := api.SELinuxOptions{}
+		v1.Convert_v1_SELinuxOptions_To_core_SELinuxOptions(tc.podSeLinux, &internalSELinuxOptions, nil)
```

Comment:
@liggitt as you can see, throughout this psp externalization, we are putting all the conversion logic into the constructor function of each strategy. so now, we list/watch external psp objects in admission controller, and do internal conversion when instantiating strategies (so that we neednt do conversion everytime we actually call strategy), finally the psp provider interface will accept internal pod/container and return internal types back to admission controllers. this will be how externalized psp works. wdyt?

Reply:
> do internal conversion when instantiating strategies (so that we neednt do conversion everytime we actually call strategy)

strategies are instantiated every time a pod is admitted. doing conversions as part of their construction is not what we want.